tests/merge/test_merge_survey_tables.py

```diff
@@ -0,0 +1,61 @@
+import pyspark.sql.functions as F
+from chispa import assert_df_equality
+
+from cishouseholds.merge import merge_survey_tables
+
+
+def test_merge_survey_tables(spark_session):
+    test_hadoop = False
+    if test_hadoop:
```

Comment:
As discussed, the code should be refactored to use a DataFrame rather than a path to hdfs. This will accommodate a reproducible test